defold-spine/src/gui_node_spine.cpp

```diff
@@ -299,6 +301,12 @@ bool SetScene(dmGui::HScene scene, dmGui::HNode hnode, dmhash_t spine_scene)
     if (!resource)
         return false;
 
+    if (node->m_FindBones)
+    {
+        node->m_FindBones = 0;
+        FindBones(node);
+    }
```

Comment:
we should find all the bones before we change scene, because new scene has its own count of bones, we don't want to override them in the next update. 

instead we find all the bones of the cloned spine node and delete them when create new bones for the new scene.

Reply:
Exactly, the new scene might have a larger number of bones, and your list hasn't changed. You just push new bones to it?

Reply:
Ah, I see, we're updating it with the _current_ list of bones, _before_ we delete them!

Reply:
yes, exactly. Just to be able to delete them